tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/BUILD

```diff
@@ -11,7 +10,7 @@ package(
 
 licenses(["notice"])  # Apache 2.0
 
-java_binary(
+minimum_java_runtime_java_binary(
```

Comment:
If it's sometimes necessary to build this for a higher bytecode version, we could keep this a regular `java_binary` and wrap it in a transitioning filegroup.

Reply:
cc @c-mita